capstone/src/main/java/servlet/Event.java

```diff
@@ -0,0 +1,28 @@
+public class Event{
```

Comment:
Add a package declaration at the top of this file:

`package com.google.sps.data;`